### ISSUE TITLE
Issue with case sensitivity on Animator/Css.elm file.  Linux has a hard time installing.

### ISSUE THREAD

**Issue by mdgriffith** [mdgriffith/elm-animator#5]
I renamed a file from `CSS.elm` to `Css.elm` at some point but the change didn't make it to the git repository because of OSX's case insensitivity.

So, this isn't a problem on OSX, but is a problem for people trying to install on a case-sensitive OS like linux.

A patch release will be out soon that will hopefully address the issue!

**Comment by monacoremo** [comment]
Awesome new package Matthew, thank you so much for releasing it! Can't wait to try it out :-)

For those on Linux that are as impatient as I am, the following command is a temporary fix:

```bash
mv ~/.elm/0.19.1/packages/mdgriffith/elm-animator/1.0.0/src/Animator/CSS.elm ~/.elm/0.19.1/packages/mdgriffith/elm-animator/1.0.0/src/Animator/Css.elm

```
Run this after the first failed install, which should look like this:

```bash
$ elm install mdgriffith/elm-animator
Here is my plan:

  Add:
    avh4/elm-color             1.0.0
    ianmackenzie/elm-units     2.4.0
    mdgriffith/elm-animator    1.0.0

Would you like me to update your elm.json accordingly? [Y/n]:
-- PROBLEM BUILDING DEPENDENCIES -----------------------------------------------

I ran into a compilation error when trying to build the following package:

    mdgriffith/elm-animator 1.0.0

This probably means it has package constraints that are too wide. It may be
possible to tweak your elm.json to avoid the root problem as a stopgap. Head
over to https://elm-lang.org/community to get help figuring out how to take this
path!th Note: To help with the root problem, please report this to the package author
along with the following information:

    avh4/elm-color 1.0.0
    elm/browser 1.0.2
    elm/core 1.0.5
    elm/html 1.0.0
    elm/json 1.1.3
    elm/time 1.0.0
    elm/virtual-dom 1.0.2
    ianmackenzie/elm-units 2.4.0

If you want to help out even more, try building the package locally. That should
give you much more specific information about why this package is failing to
build, which will in turn make it easier for the package author to fix it!

```

This caches the results in `~/.elm`, where we can apply the little fix...

**Comment by mdgriffith** [comment]
I just published `v1.0.1` where the name should be cased correctly.  @monacoremo would you mind trying to do an install now to help me verify it's resolved?

**Comment by mdgriffith** [comment]
I got confirmation that it works 🎉
